microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -75,8 +91,12 @@ This release includes integration with the greenboot health check framework. Gre
 //[id="microshift-4-13-deprecated-removed-features"]
 //== Deprecated and removed features
 
-//[id="microshift-4-13-bug-fixes"]
-//== Bug fixes
+[id="microshift-4-13-bug-fixes"]
+== Bug fixes
+
+* Previously, {product-title} generated `kubeconfig` files using certificate authorities (CA) shared by all embedded components. With this update, the CAs and certificates are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-10223[*OCPBUGS#10223*])
+
+* Previously, the `systemd-resolved` configuration was used by kubelet as an alternative dns resolver, but the DNS Corefile configuration did not use the same `systemd-resolved` configuration. With this update, {product-title} uses network name resolution provided by `systemd-resolved` for both kublet and cluster DNS when `systemd-resolved.service` is enabled. (link:https://issues.redhat.com/browse/OCPBUGS-6786[*OCPBUGS-6786*])
```

Comment:
s/dns resolver/DNS resolver
s/uses network name/uses the network name  (Not 100% sure on this one)
s/when `systemd-resolved.service`/when the `systemd-resolved.service`